website/versioned_docs/version-0.79/getting-started.md

```diff
@@ -0,0 +1,135 @@
+---
+id: version-0.79-getting-started
```

Comment:
Same comment as above. this should just change the getting-started document in the docs folder